### DISCUSSION TITLE
Le pattern Builder

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#175] [Announcements]
# Le pattern Builder

Nous allons voir dans quelles situations on utilise le pattern Builder et comment l'implémenter en Java.

https://zerofiltre.tech/articles/261

**Comment by imphilippesimo** [comment]
Comment lister les précedents articles de la même série : 

Nous continuons notre série sur les design patterns(patrons de conception) dont les articles précédents sont les suivants : 

* [Design pattern Memento](https://zerofiltre.tech/articles/266)
* [Design pattern Stratégie](https://zerofiltre.tech/articles/259) 
* -//-//- yyyy

Aujourd'hui nous allons voir un autre pattern, le modèle Builder.  🏗️

**Comment by imphilippesimo** [comment]
>Supposons que nous voudrions implémenter la fonctionnalité avec du code. Pour se faire, nous allons  

J'aurais dit  

> Pour implémenter cette fonctionnalité, nous allons

**Comment by imphilippesimo** [comment]
> Test
C'est plus une exécution qu'un test :) 

> Exécution

**Comment by imphilippesimo** [comment]
> Il faut tout de même veiller à ne l'utiliser que lorsque c'est pertinent pour bien profiter de ses avantages.    

Ex: Vous avez plus de 3 représentations
